recipe/meta.yaml

```diff
@@ -18,16 +19,16 @@ requirements:
     - autoconf
     - m4
     - perl
```

Comment:
Would recommend pinning this to some exact version of `perl` and re-rendering to get rid of the `perl` matrix that we don't need here.